ExerciseTwo/exercise.html

```diff
@@ -1,48 +1,36 @@
-<!-- 
-    INSTRUCTIONS:
-    Fix the 24 lines with indentation issues below.
--->
-    <!DOCTYPE html>
-<head>
+<!DOCTYPE html>
+    <html>
```

Comment:
This should be at the same level of indentation as line 1.
Please adjust any and all indentation that would be affected by this below as well.